composer.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "fusesource/stomp-php",
     "description": "stomp support for PHP",
+    "abandoned": true,
```

Comment:
That's good, didn't see this the last time I was looking for a way to setup a proper `abandoned` handling.